apps/server/test/lexical/server/code_intelligence/entity_test.exs

```diff
@@ -480,6 +480,215 @@ defmodule Lexical.Server.CodeIntelligence.EntityTest do
     end
   end
 
+  describe "struct resolve/2" do
```

Comment:
Another point I think we can improve is showing default values for structs in the struct section.

Of course, addressing this in a future PR is fine.

Reply:
> Another point I think we can improve is showing default values for structs in the struct section.

I'd like to do this! I'm thinking separate PR though, because I think the nicest way to do this is to merge the defaults and the type spec, so you see something like:

```elixir
%MyModule{
  foo: :default_foo :: atom(),
  bar: "baz" :: MyModule.custom_type()
}
```

and since that's non-trivial and has some edge cases, IMO best in another PR.

Reply:
or, format like this: 

```elixir
%MyModule{
  foo: atom() \\ :default_foo,
  bar: MyModule.custom_type() \\ "baz"
}
```

Doing this formatting should have some work, feel free to do this at another PR